docs/api/structures/service-worker-info.md

```diff
@@ -0,0 +1,5 @@
+# ServiceWorkerInfo Object
+
+* `scriptUrl` String - The full URL to the script that this service worker runs
+* `scope` String - The base URL that this service worker is active for.
+* `renderProcessId` Number - The virtual ID of the process that this service worker is running in.  This is not an OS level PID.  This aligns with the ID set used for `webContents.getProcessId()`.
```

Comment:
If we wanted to add e.g. methods for killing / restarting service workers, would they go on this object? e.g.

```
session.defaultSession.serviceWorkers.getAllRunning().forEach(sw => sw.shutdown())
```

or something else?

Reply:
Either that, or have `serviceWorkers.stopByVersionId(versionid)`